### ISSUE TITLE
Bug with reading calculation group table details from Power BI Desktop?

### ISSUE THREAD

**Issue by samaguire** [TabularEditor/TabularEditor3#441]
Maybe this is just a quirk with Power BI Desktop, but the below screen clip shows where I've made some changes to a calculation group in Desktop however these changes aren't reflected in TE3 after a refresh/revert. I've added a calculated column and changed the formats.

![image](https://user-images.githubusercontent.com/62320770/155457600-d5113fc3-81d5-489d-92ef-70515e877346.png)

**Comment by otykier** [comment]
@samaguire what happens if you save the model in Desktop, then close Desktop and restart, reloading the same model. Is the calculated column still present on the calc group table then? Does it then appear in Tabular Editor?

**Comment by samaguire** [comment]
> @samaguire what happens if you save the model in Desktop, then close Desktop and restart, reloading the same model. Is the calculated column still present on the calc group table then? Does it then appear in Tabular Editor?

Okay, so I didn't keep the throw away file I created to double check what I was seeing in the productionised dataset I was working on was an issue. So couldn't do this for you. However, the original dataset I noticed this in is displaying correctly, and I haven't been able to recreate this in a clean PBIX file.

I'm going to close this and chalk it up to weird and potentially fixed with the latest Desktop update (which I installed after this ticket was raised).
